src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -166,6 +166,49 @@ tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
   return deriv_spatial_metric;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
```

Comment:
What is the reasoning behind this ordering of the template parameters? It might be nice to keep it consistent with the tensor's parameters, i.e. `<DataType, SpatialDim, Frame>`.

Reply:
Oh, I guess this is how the rest of the file orders these.. :-/

Reply:
yeah, not just this file but all code in evolution system GeneralizedHarmonic follows this ordering, which is the precedent i followed

Reply:
Definitely the right choice for this PR then 👍